project1/ImmersiveView.swift

```diff
@@ -57,20 +58,23 @@ class SequenceViewModel: ObservableObject {
         self.particleSystem.emitterShape = .sphere
         self.particleSystem.birthLocation = .volume
         self.particleSystem.birthDirection = .normal
-        self.particleSystem.emitterShapeSize = [10, 10, 10] * 0.05
+        self.particleSystem.emitterShapeSize = [10, 10, 10] * Float.random(in: 0.05...0.80)
```

Comment:
This is fine for now, but I think it would make a lot of sense to use a function that returns a JSON object that holds all of the configurations. Then we can separate the concern of experimenting with values from the infrastructure that maintains the particle system life cycle. That would make for cleaner experimentation while mitigating the risk of accidental regressions. I have already implemented that in my code that I have not yet re-submitted I can create a PR for that later.